generator/wrapper_gen.py

```diff
@@ -164,22 +164,92 @@ def __dealloc__(self):
     cdef void c_enable_thread_pinning(bool enabled) except +
 
 
-def daalinit(nthreads = -1):
+def daalinit(nthreads: int = -1) -> None:
+    '''
+    Set number of threads for daal4py
+
+    This modifies the number of threads configured for daal4py, which is a
+    global setting.
```

Comment:
Mentioning what does a global scope mean might add clarity, applicable for other functions too:
```suggestion
    This modifies the number of threads configured for daal4py, which is a
    global setting per a process.
```

Reply:
Added a more verbose explanation.